app/views/layouts/_adv_search_body.html.haml

```diff
@@ -79,7 +79,7 @@
             %label.control-label.col-md-5
               = _("Global search:")
             .col-md-6
-              - checked = @edit && @edit[:expression] && @edit[:expression][:selected] && @edit[:expression][:selected][:typ] == "global"
+              - checked = @edit && @edit[:expression] && @edit[:expression][:selected] && @edit[:search_type] == "global"
```

Comment:
@isimluk I still don't quite know where that comes from (since `typ` is never in the DB) .. but definitely yes, it does make sense to keep all the info in one place.

(Though, I think it should be `@edit[:expression][:selected][:search_type]` to match the db)